Automattic/jetpack
# AI Assistant: PoC scroll on suggestion stream

The overall consensus with AI UIs seems to be that of a chat history.

Fixes #36506 

## Proposed changes:
This PR works as a proof of concept on the scrolling behavior while the AI suggestion is streaming. The scroll is triggered on any suggestion chunk so to always have the latest streamed content in view.

### Other information:


## Jetpack product discussion
p1711048313305849-slack-C054LN8RNVA

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Go to the editor and insert an AI Assistant block. For optimal testing you may make your browser window shorter. Ask AI assistant for some content you know will exceed the height of the viewport.
As the suggestions starts coming and the end of the viewport is reached, the editor should scroll to keep the bottom of the block (latest block content inserted) in view.
